apps/aragon-fundraising/app/src/screens/Reserves.js

```diff
@@ -221,123 +175,114 @@ export default () => {
     }
   }
 
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const editMonthlyAllocationButton = <Button icon={<img src={EditIcon} />} label="Edit monthly allocation" onClick={() => setOpened(true)}></Button>
+
   return (
-    <ContentWrapper>
-      <div className="settings">
-        <h1 className="title bold">Edit reserve settings</h1>
-        <div className="settings-content">
-          <div css="margin-right: 4rem;">
-            <div css="display: flex; flex-direction: column; margin-bottom: 1.5rem;">
-              {NotificationLabel('Monthly allocation', hoverTextNotifications[0])}
-              <Text as="p" css="padding-right: 1.5rem; font-weight: bold;">
-                {displayRate} DAI / month
-              </Text>
-            </div>
-            <div css="display: flex; flex-direction: column; margin-bottom: 1.5rem;">
-              {NotificationLabel('Floor', hoverTextNotifications[2])}
-              <Text as="p" css="padding-right: 1.5rem; font-weight: bold;">
-                {displayFloor} DAI
-              </Text>
+    <React.Fragment>
```

Comment:
The linter prefers the shorthand version of Fragments `<></>`

Reply:
Done in https://github.com/AragonBlack/fundraising/pull/114/commits/7fefdbb7c7f70f62b47cc6d0087481f72cd61c2d